dispatch-client/src/main/scala/org/htt4s/client/dispatch/DispatchClient.scala

```diff
@@ -0,0 +1,50 @@
+package org.htt4s.client.dispatch
+
+import dispatch._, Defaults._
+import com.ning.http.client.{Response => DResponse}
+
+import org.http4s._
+import org.http4s.client._
+
+import scala.collection.JavaConverters._
+import scalaz.concurrent.Task
+
+import scalaz._
+import Scalaz._
+
+object DispatchClient {
+  def apply(): Client = {
+    Client(Service.lift { req =>
+      Task.async[DResponse] { register =>
+        Http(toDRequest(req)).onComplete {
```

Comment:
We have a [utility to convert a `Future` to a `Task`](https://github.com/http4s/http4s/blob/master/core/src/main/scala/org/http4s/util/Task.scala#L19).